README.md

```diff
@@ -262,6 +262,47 @@ jobs:
       - run: rebar3 ct
 ```
 
+### Compatibility
+
+One can run matrix-style testing but with pinned compatible versions in the following manner:
+
+```yaml
+  test:
+    runs-on: ${{matrix.erlang.os}}
+    name: Erlang/OTP ${{matrix.erlang.otp}} / rebar3 ${{matrix.erlang.rebar3}}
+    strategy:
+      matrix:
+        erlang:
```

Comment:
This list will very easily get outdated. To ease management (and reduce the size) of the README, could you remove everything below 26?